### DISCUSSION TITLE
Resolutions support

### DISCUSSION THREAD

**Discussion by 1bionix1** [AstralVixen/GeForce-Infinity#5] [General]
Hi! Your project sounds promising! I would like to know if you are planning or if its even possible to add more resolution support in GFN.

**Comment by AstralVixen** [comment]
Hey! Could you please clarify a bit?
Because what I got from this question was that if I'm able to add more in-game resolution settings? Because if that was the question, sadly it's not possible.

**Reply by 1bionix1** [reply]
Like in windows app and android app, we have higher resolutions. Wondering if it's possible to mimic the android app. 

![Screenshot_20241214-175651](https://github.com/user-attachments/assets/a31d7686-7148-4491-a2e3-990ac92fa5b5)

**Comment by AstralVixen** [comment]
I will do some research into this and try a few things but I don't really think it's that possible, as this it the web based version since it's an electron app. Thank you for the suggestion tho and maybe I'll be able to figure something out

**Reply by 1bionix1** [reply]
Glad to help! Keep us posted on your findings 🙂
